src/awesome/root.rs

```diff
@@ -93,6 +95,29 @@ fn tags<'lua>(lua: &'lua Lua, _: ()) -> rlua::Result<Table<'lua>> {
     Ok(table)
 }
 
+/// Get or set global key bindings.
+///
+/// These bindings will be available when you press keys on the root window.
+fn root_keys<'lua>(lua: &'lua Lua, key_array: rlua::Value) -> rlua::Result<rlua::Value<'lua>> {
+    match key_array {
+        // Set the global keys
+        Value::Table(key_array) => {
+            lua.globals().set(ROOT_KEYS_HANDLE, key_array)?;
```

Comment:
How about `set_named_registry_value` instead of saving this in the global table?

Reply:
Never mind, the next commit seems to do just that